libc/src/setjmp/x86_64/CMakeLists.txt

```diff
@@ -8,12 +8,6 @@ add_entrypoint_object(
     libc.hdr.types.jmp_buf
   COMPILE_OPTIONS
     -O3
```

Comment:
It seems weird to hardcode the optimization level, but that seems to be llvm-libc style so OK.

Reply:
Yes, I've discussed this with the team a bit.  I'd prefer to "just do what libc++ does in terms of configuring the build for size vs performance."  Looking at [their docs](https://releases.llvm.org/19.1.0/projects/libcxx/docs/BuildingLibcxx.html) it seems that [CMAKE_BUILD_TYPE](https://releases.llvm.org/19.1.0/projects/libcxx/docs/BuildingLibcxx.html#cmake-options) is perhaps the only way to signal that you'd like a release build, vs MinSizeRel.

Either way, we should file a new issue and discuss this [there](https://github.com/llvm/llvm-project/issues/112409).